subsys/net/l2/wifi/wifi_shell.c

```diff
@@ -1273,7 +1282,8 @@ SHELL_STATIC_SUBCMD_SET_CREATE(wifi_commands,
 		    "[-a, --dwell_time_active <val_in_ms>] : Active scan dwell time (in ms) on a channel. Range 5 - 1000 ms.\n"
 		    "[-p, --dwell_time_passive <val_in_ms>] : Passive scan dwell time (in ms) on a channel. Range 10 - 1000 ms.\n"
 		    "[-s, --ssid <Comma separate list of SSIDs>] : SSID list to scan for.\n"
-		    "[-m, --max_bss <val>] : Max BSSIDs to scan for. Range 1 - 65535.",
+		    "[-m, --max_bss <val>] : Max BSSIDs to scan for. Range 1 - 65535.\n"
+		    "[-c, --chan <Comma separated list of channel ranges>] : Channels to be scanned. The channels have to be specified in the form band1:chan1,chan2_band2:chan3,..etc. The chan has to be a list of comma separated values where each value is either a single channel or a channel range specified as chan_start-chan_end. Each band channel set has to be separated by a _. E.g. a valid channel specification can be 2:1,6-11,14_5:36,149-165,44",
```

Comment:
isn't `space` a better delimiter than `_`?

Reply:
`space` is resulting in a separate argv, hence need a different delimiter.